pkg/preparer/orchestrate_test.go

```diff
@@ -113,25 +135,10 @@ func TestPreparerWillNotLaunchIfSHAIsTheSame(t *testing.T) {
 		currentManifest: testManifest,
 	}
 
-	var p *Preparer
+	p := testPreparer(testManifest)
 	success := p.installAndLaunchPod(testManifest, testPod, logging.DefaultLogger)
 
 	Assert(t).IsTrue(success, "Should have been a success to prevent retries")
 	Assert(t).IsFalse(testPod.launched, "Should not have attempted to launch")
 	Assert(t).IsTrue(testPod.installed, "Should have installed")
 }
-
-func TestInstallReturnsFalseIfManifestErrsOnRead(t *testing.T) {
```

Comment:
Hmm, we could totally keep this -- you see how you had `func (f *FakeStore) Pod(string, string) (*pods.PodManifest, error)` return a manifest of your choosing. So you could make it return an error of your choosing as well.